core/string/ustring.cpp

```diff
@@ -240,6 +240,70 @@ String String::word_wrap(int p_chars_per_line) const {
 	return ret;
 }
 
+Error String::parse_url(String &r_scheme, String &r_host, int &r_port, String &r_path) const {
```

Comment:
Might be worth adding a short description similar to what you put in the PR, since it's not exposed and thus documented in the classref.